### DISCUSSION TITLE
Information Density & Visual Breaks

### DISCUSSION THREAD

**Discussion by rmaspero** [rails/rails#51416] [Docs]
Hey, Absolutely love the fresh new look 🎉 

We discussed the new docs as a team this morning. I thought I'd share some of the feedback our developers had. Generally they loved the much cleaner design and more modern look and feel.

We noticed the body font size seems larger and there is a lot of white space. This means when compared to the old guides the information density was lower. Aka we had to scroll a lot more to see things and it was harder to fit two code examples on the same page. A lot of our dev team ended up looking at the guides zoomed out.

Reducing some of the heavy white spacing and dropping the font size of the body text might help here.

We also found the large amounts of "white" whitespace and lack of visual dividers made it easy to get lost e.g. the blue section at the top of each guide made it clear where the guide started or the header being red meant we knew where the navigation stopped. Wonder if a pop of colour or some backgrounds might help guide the readers.

Don't know if this is a bug or a design thing but we noticed the numbers in the guide headings were a different font weight to the titles.

Honestly we love the design and we are just being picky, so @AmandaPerino and the team behind it all congrats.

**Comment by AmandaPerino** [comment]
Hi @rmaspero thanks for submitting this feedback - please pass on our thanks to the team! No such thing as too picky. :)
We have heard the same feedback about and information & text density feedback, so we are currently taking that into consideration in this next round of updates.
Some of these changes are deliberate design (chapter numbers versus titles), and some might just take some getting used to after 15 years, but we're gathering all feedback and seeing what changes we should make.
Thanks again!

**Reply by mweitzel** [reply]
I was wondering why the main content text didn't respect my system default as well. I prefer to use the site on 90% on desktop to get it back from 18px back closer to 16px, but that isn't an option on mobile. This is a very noticeable backwards step on small laptops as well.

I totally understand wanting a "cleaner and sleeker look" but docs are intended to present information. Reducing visual noise is one thing, but reducing the amount of actual information on the page at a time is another.

**Reply by AmandaPerino** [reply]
Hi @mweitzel See above: text and information density is being addressed. Thank you!

**Comment by AmandaPerino** [comment]
Hi @rmaspero & @mweitzel - updates were made in #51444 to address some of these points:
- a line divider is now the visual cue between intro and content (replacing the blue border)
- font reduced to 16px
Going to close this out now but if you have any other feedback please feel free to open a new discussion and tag me. Thanks again!
